streams/src/main/java/org/apache/kafka/streams/processor/assignment/assignors/StickyTaskAssignor.java

```diff
@@ -173,6 +178,7 @@ private static void assignStandby(final ApplicationState applicationState,
                                       final AssignmentState assignmentState) {
         final Set<TaskInfo> statefulTasks = applicationState.allTasks().values().stream()
             .filter(TaskInfo::isStateful)
```

Comment:
you can remove this line actually, the changelogs check is sufficient (not to mention I just realized this filter is probably broken at the moment anyways since we still need to fix the `stateStoreNames` thing)